vrecord

```diff
@@ -1130,7 +1160,7 @@ while [[ "${RUNTYPE}" = "edit" ]] ; do
     _edit_mode
 done
 
-if [[ "${INVERT_PHASE}" = 1 ]] ; then
+if [[ "${INVERT_PHASE}" = 'true' ]] ; then
```

Comment:
Why single quotation marks (`'`) and not double quotation marks (`"`) here?

Reply:
There's no content to interpolate so `'true'` and `"true"` would be identical in this case.

Reply:
In the script, there are 3 conditions to test the value of $INVERT_PHASE, but only 2 are changed, so the commit only partly fixes the issue.

Reply:
>There's no content to interpolate so 'true' and "true" would be identical in this case.

I know that (and in past I used to distinguish this myself!), but I was told that today it’s considered good practice to use only double quotation marks in Bash. We have many other places in the code where no interpolation occurs.

Reply:
I don't want to block the merge, so I'll stall the merge for a bit to allow the contributor to have a chance to consider, else consider a subsequent pr.

Reply:
Oh, my remark it’s not blocking at all! I was just wondering why here.

Reply:
Style-wise I prefer to use `'` in situations like this where there is no interpolation as I feel that it helps distinguish in the code where things like variable substitution is actually occurring, as well as forcing me to be more conscious about differences between quotes in their different applications (such as quotes in quotes).

Reply:
Then I will open a PR to systematise the different use of `'` and `"`.